src/sp800_108.rs

```diff
@@ -367,6 +373,8 @@ impl Sp800Operation {
         }
     }
 
+    /* NOTE: In this function the len is intnetionally truncated by
```

Comment:
```suggestion
    /* NOTE: In this function the len is intentionally truncated by
```
The same in the previous block.